lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc

```diff
@@ -305,23 +305,30 @@ function paraneue_dosomething_preprocess_node(&$vars) {
  * @see paraneue_dosomething_preprocess_node().
  */
 function paraneue_dosomething_add_info_bar(&$vars) {
-  // Initialize info_bar_vars with variable that will always be set:
-  $zendesk_form_header = t(variable_get('dosomething_zendesk_form_header'));
-  $info_bar_vars = array(
-    'zendesk_form_header' => $zendesk_form_header,
-  );
-  // List $vars variable names to pass to info bar:
-  $info_var_names = array(
-    'formatted_partners',
-    'zendesk_form',
+  // Initialize info_bar_vars with a variable that will always be set:
```

Comment:
Maybe remove this comment cause it's a little confusing since it doesn't apply anymore to the code immediately below it... if the `formatted_partners` isn't set then the `info_bar_vars` _doesn't_ get a vlaue w/ a var that will always be set...